recipe/run_conda_forge_build_setup_osx

```diff
@@ -12,7 +12,7 @@ export PYTHONUNBUFFERED=1
 #   way is working well enough.
 export MACOSX_DEPLOYMENT_TARGET="10.9"
 
-if [ "$CIRCLECI" == "true" || "$OSX_FORCE_SDK_DOWNLOAD" == "1" ]; then
+if [[ "$CIRCLECI" == "true" || "$OSX_FORCE_SDK_DOWNLOAD" == "1" ]]; then
```

Comment:
Should be fine. POSIX-y alternative would be
```sh
if [ "$CIRCLECI" == "true" ] || [ "$OSX_FORCE_SDK_DOWNLOAD" == "1" ]; then
```
IDK if there is any preference for the one or the other here, but I guess it shouldn't matter since we have `#!/bin/bash` anyway.

Can you also bump the version number?

Reply:
> Can you also bump the version number?

Done!

Reply:
Yes, the current `[ ... || ... ]` is broken.
`[[ ... || ... ]]` or `[ ... ] || [ ... ]` are the working alternatives is what I meant to say :wink:.

Reply:
Doesn't look like CircleCI was using @mbargull's suggestion. That said, happy to go with the current fix (we force `bash` in this script anyways 😉).

Reply:
Hard to read the diff when your eyes are trained to the the first and last bracket ;-p